app/src/main/kotlin/com/passwordstore/android/utils/Constants.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright © 2014-2019 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+package com.passwordstore.android.utils
+
+object Constants {
+    const val IS_FIRST_RUN = "is_first_run"
```

Comment:
This is a preference key, right? Rename the class to match that, something like `PrefKeys` or `PreferenceKeys` alongwith a comment at the top of the class that makes it absolutely clear for any future contributors.

Reply:
Sure